config/samples/sample-mpijob.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: kubeflow.org/v2beta1
```

Comment:
I'm considering whether we can move stuff like this to a top level folder like `examples`, which is easy to find.

Reply:
Yea maybe post an issue.  I think that makes a sense to me as I usually think to go to examples to understand what the object looks like

Reply:
https://github.com/kubernetes-sigs/kueue/issues/812 posted!